evergreen/run-tests.sh

```diff
@@ -92,36 +92,47 @@ fi
 echo "Running $AUTH tests over $SSL for $TOPOLOGY with $COMPRESSOR compressor and connecting to $MONGODB_URI"
 
 if [[ "$OS" =~ Windows|windows ]]; then
-  if [[ "$FRAMEWORK" == "nil" ]]; then
-    export TARGET="Test"
-  else
-    export TARGET="Test${FRAMEWORK}"
-  fi
+  export TARGET="Test${FRAMEWORK}"
   if [ "$OCSP_TLS_SHOULD_SUCCEED" != "nil" ]; then
     export TARGET="TestOcsp"
     certutil.exe -urlcache localhost delete # clear the OS-level cache of all entries with the URL "localhost"
   fi
 else
-  export TARGET="Test"
+  export TARGET="Test${FRAMEWORK}"
 fi
 
 echo "Final MongoDB_URI: $MONGODB_URI"
 if [ "$TOPOLOGY" == "sharded_cluster" ]; then
   echo "Final MongoDB URI with multiple mongoses: $MONGODB_URI_WITH_MULTIPLE_MONGOSES"
 fi
 for var in TMP TEMP NUGET_PACKAGES NUGET_HTTP_CACHE_PATH APPDATA; do
-  export $var=z:\\data\\tmp;
+  if [ "$OSTYPE" = "cygwin" ]; then
```

Comment:
because of this comment https://github.com/durran/mongo-csharp-driver/pull/2#discussion_r526987531, should we use `$OS`?

Reply:
Done